modules/component/types/extractValueTypeFromActionWithDefault.ts

```diff
@@ -0,0 +1,22 @@
+import {LActionValueForType} from './extractValueTypeFromAction'
+
+/**
+ * Infer type of values from action set given action type's type with fallback type
+ *
+ * ```typescript
+ * import {Action} from '@action-land/core'
+ *
+ * LActionValueForTypeWithDefault<Action<10, 'a'> | Action<{count: 10}, 'b'>, 'c', 'default'> // outputs  'default'
+ * ```
+ *
+ * @typeparam A Set of actions
```

Comment:
- `Set of actions` means what? Should it simply be `union type`?
- set of actions makes me think of a `List` action